src/devices/src/virtio/persist.rs

```diff
@@ -43,6 +43,10 @@ pub struct QueueState {
 
     next_avail: Wrapping<u16>,
     next_used: Wrapping<u16>,
+
+    /// The number of added used buffers since last guest kick
+    #[version(start = 2)]
+    num_added: Wrapping<u16>,
```

Comment:
You should also update the version map to account for this new QueueState version.

Reply:
Done.